clients/client-java/src/main/java/com/datastrato/gravitino/client/RelationalCatalog.java

```diff
@@ -379,24 +379,15 @@ public Schema alterSchema(NameIdentifier ident, SchemaChange... changes)
   @Override
   public boolean dropSchema(NameIdentifier ident, boolean cascade) throws NonEmptySchemaException {
```

Comment:
Will it only throw the NonEmptySchemaException?

Reply:
Other exceptions will also be thrown, but it's not explicitly declared.